frontend/src/components/navbar/Navbar.tsx

```diff
@@ -8,6 +8,7 @@ import { useHistory } from 'react-router-dom';
 
 function Navbar() {
 
+  const MAX_SCREEN_WIDTH = 950;
```

Comment:
I like this practice!! Let's all start doing this with predefined values like this @jersteve @alexandrofuentelzas